library/core/src/sync/atomic.rs

```diff
@@ -155,6 +155,7 @@ pub struct AtomicBool {
 #[stable(feature = "rust1", since = "1.0.0")]
 impl Default for AtomicBool {
     /// Creates an `AtomicBool` initialized to `false`.
+    #[inline]
     fn default() -> Self {
```

Comment:
Isn't there another on on `AtomicPtr` as well?

Reply:
`AtomicPtr` is generic, which means it's already cross-crate inlineable, right?

Reply:
Nevermind, that one is the only generic one.